_data/rules.yml

```diff
@@ -391,6 +391,32 @@ groups:
                 severity: critical
                 for: 2m
 
+      - name: VMware
+        exporters:
+          - name: pryorda/vmware_exporter
+            doc_url: https://github.com/pryorda/vmware_exporter
+            rules:
+              - name: Virtual Machine Memory Warning
+                description: 'High memory usage on {{ $labels.instance }}: {{ $value | printf "%.2f"}}%'
+                query: '((vmware_vm_mem_usage_average / 100) >= 90) and ((vmware_vm_mem_usage_average / 100) < 95)'
+                severity: warning
+                for: 30m
+              - name: Virtual Machine Memory Critical
+                description: 'High memory usage on {{ $labels.instance }}: {{ $value | printf "%.2f"}}%'
+                query: '((vmware_vm_mem_usage_average / 100) >= 95)'
+                severity: critical
+                for: 5m
+              - name: High Number of Snapshots
+                description: "High snapshots number on {{ $labels.instance }}: {{ $value }}"
+                query: '(vmware_vm_snapshots < 3)'
```

Comment:
You mean `> 3` ?

Reply:
Should we consider snapshots as a bad thing?

I'm pretty sure we should track the space left on disk instead. WDYT?

Reply:
Will update this as a warning and set to > 3; however, per VMware [documentation](https://kb.vmware.com/s/article/1025279), "Maximum of 32 snapshots are supported in a chain. However, for a better performance use only 2 to 3 snapshots." 
Disk would also be a good metric to base this off of; however, until I play more with the exporter and tune things a bit I didn't want to stray far from the exporters [alerts](https://github.com/pryorda/vmware_exporter/blob/master/alerts/vmware.rules.yml).